mobile/src/navigation/app/common/constants/constants.ts

```diff
@@ -192,6 +166,18 @@ const NAVIGATION_ITEMS: DrawerNavigationItem[] = [
     permissions: [],
     isAuthRequired: true,
   },
+  {
+    name: AppScreenName.COURSE,
+    component: Course,
+    permissions: [],
+    isAuthRequired: false,
+  },
+  {
+    name: AppScreenName.COURSE_MODULE,
+    component: CourseModule,
+    permissions: [],
+    isAuthRequired: true,
+  },
```

Comment:
CourseModule navigation should be nested in Course navigation (after you finish with all screens, only items with drawerGroup property should remain here, all other screens should be nested)